src/applications/personalization/profile/mocks/endpoints/user/index.js

```diff
@@ -99,7 +99,7 @@ const baseUserResponses = {
           servedInMilitary: true,
         },
         inProgressForms: [],
-        prefillsAvailable: ['21-686C'],
+        prefillsAvailable: ['21-686C', 'mock-form-ae-design-patterns'],
```

Comment:
lets remove this change from the profile code, because we now have our own mock server folder set up in our app.

Reply:
done